vismne/viz.py

```diff
@@ -1,36 +1,77 @@
 """Need doc."""
-from .io import _get_subjects_dir
-from .utils import Surface, _check_units
+import os
+
+import numpy as np
+
+from vispy import scene
+
+from .io import _get_subjects_dir, read_scalar_data
+from .utils import Surface, _check_units, string_types
 from .visuals import BrainMesh
 
 
-lh_viewdict = {'lateral': {'v': (180., 90.), 'r': 90.},
-               'medial': {'v': (0., 90.), 'r': -90.},
-               'rostral': {'v': (90., 90.), 'r': -180.},
-               'caudal': {'v': (270., 90.), 'r': 0.},
-               'dorsal': {'v': (180., 0.), 'r': 90.},
-               'ventral': {'v': (180., 180.), 'r': 90.},
-               'frontal': {'v': (120., 80.), 'r': 106.739},
-               'parietal': {'v': (-120., 60.), 'r': 49.106}}
-rh_viewdict = {'lateral': {'v': (180., -90.), 'r': -90.},
-               'medial': {'v': (0., -90.), 'r': 90.},
-               'rostral': {'v': (-90., -90.), 'r': 180.},
-               'caudal': {'v': (90., -90.), 'r': 0.},
-               'dorsal': {'v': (180., 0.), 'r': 90.},
-               'ventral': {'v': (180., 180.), 'r': 90.},
-               'frontal': {'v': (60., 80.), 'r': -106.739},
-               'parietal': {'v': (-60., 60.), 'r': -49.106}}
+lh_viewdict = {'lateral': {'v': (-90., 0.), 'r': 90., 'xyz': [1, 2]},
+               'medial': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': -180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (120., 80.), 'r': 106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-120., 60.), 'r': 49.106, 'xyz': [0, 1]}}
+rh_viewdict = {'lateral': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'medial': {'v': (-90., 0.), 'r': 90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': 180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (60., 80.), 'r': -106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-60., 60.), 'r': -49.106, 'xyz': [0, 1]}}
 viewdicts = dict(lh=lh_viewdict, rh=rh_viewdict)
 
 
 class Brain(object):
-    """docstring for Brain."""
-
-    def __init__(self, subject_id, hemi, surf, title=None, views=['lat'],
-                 offset=True):
+    """Class for visualizing a brain using multiple views in vispy.
+
+    Parameters
+    ----------
+    subject_id : str
```

Comment:
ideally you would like to have a `figure` parameter so that the brain plot can be updated by the user easily ... I think you mentioned why you didn't have it, but now I forgot :)